di-ipv-core-stub/src/main/java/uk/gov/di/ipv/stub/core/utils/HandlerHelper.java

```diff
@@ -207,6 +207,7 @@ public String getUserInfo(
                 state,
                 userInfoRequest.getURL());
         HTTPResponse userInfoHttpResponse = sendHttpRequest(userInfoRequest);
+        LOGGER.info("UserInfoHttpResponse : {}", userInfoHttpResponse.getContent());
```

Comment:
Should this be debug it could be quite heavy on the logging

Reply:
Changed to debug level.